qiskit_experiments/calibration_management/basis_gate_library.py

```diff
@@ -407,14 +427,20 @@ def _build_schedules(self, basis_gates: Set[str]) -> Dict[str, ScheduleBlock]:
         if "cr45m" in basis_gates:
             with pulse.build(name="cr45m") as cr45m:
                 pulse.play(
-                    pulse.GaussianSquare(cr_dur, -cr_amp, risefall_sigma_ratio=cr_rf, sigma=sigma),
+                    pulse.GaussianSquare(
+                        cr_dur, -cr_amp, angle=cr_angle, risefall_sigma_ratio=cr_rf, sigma=sigma
+                    ),
                     u_chan,
                 )
 
                 if self._target_pulses:
                     pulse.play(
                         pulse.GaussianSquare(
-                            cr_dur, -tgt_amp, risefall_sigma_ratio=cr_rf, sigma=sigma
+                            cr_dur,
+                            -tgt_amp,
```

Comment:
Alternatively we can `amp=tgt_amp` and `angle=tgt_angle+pi`. What should be the standard notation in our package? I tend to prefer updating the angle.

Reply:
I prefer that too.